### PR TITLE
Fixes #64 - Add missing UA-Full-Version

### LINKED ISSUE #64: Sec-CH-UA-Full-Version seems to be missing from the spec.
Hi

In Section 7.1 of the spec https://wicg.github.io/client-hints-infrastructure/

It says...
A client hints token is a byte-lowercase representation of one of Save-Data, DPR, Width, Viewport-Width, Viewport-Height, Device-Memory, RTT, Downlink, ECT, Sec-CH-UA-Arch, Sec-CH-UA-Bitness, Sec-CH-UA-Model, Sec-CH-UA-Platform, Sec-CH-UA-Platform-Version, Sec-CH-UA or Sec-CH-UA-Mobile.

But **Sec-CH-UA-Full-Version** is missing from the list.

Also not mentioned in Section 7.2 or 7.4 either.
Nor in the Section 9 Index.

### PR DESCRIPTION
**Pull request by miketaylr**
PTAL @yoavweiss

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -292,13 +292,14 @@ A <dfn export>client hints token</dfn> is a [=byte-lowercase=] representation of
   `RTT`,
   `Downlink`,
   `ECT`,
+  `Sec-CH-UA`,
```
Comment (miketaylr) [inline index.bs:295]: I also alphabetized the UA stuff, hopefully not too annoying to review.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by miketaylr** [comment]
thxx

**Comment by miketaylr** [comment]
@yoavweiss FYI, i landed this with https://github.com/WICG/client-hints-infrastructure/pull/69/commits/d34b31b442c60a8d6bd22305d769c3de362f32bb

**Pull request merged**